### PR TITLE
swift-format 0.50700.1

### PR DESCRIPTION
**Pull request by jzbrooks**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Package.swift:
    expected replacement of ".upToNextMinor(from: \"0.50700.0\")" with ".exact(\"0.50700.0\")"
```

**Comment by jzbrooks** [comment]
@chenrui333, this test failure came up in https://github.com/Homebrew/homebrew-core/pull/119766#issuecomment-1376407152, also. It appears to be an issue with the test setup, not the formula.

File: Formula/swift-format.rb [hunk 1]
```diff
@@ -26,14 +25,14 @@ class SwiftFormat < Formula
...
-  depends_on xcode: ["14.0", :build]
+  depends_on xcode: ["14.2", :build]
 
   uses_from_macos "swift"
 
   def install
     # Support Swift 5.7.
     # Remove when minimum supported Swift >= 5.7.1.
-    inreplace "Package.swift", '.upToNextMinor(from: "0.50700.0")', '.exact("0.50700.0")' if OS.mac? && build.stable?
+    inreplace "Package.swift", '.upToNextMinor(from: "0.50700.1")', '.exact("0.50700.1")' if OS.mac? && build.stable?
```
Comment (SMillerDev) [inline Formula/swift-format.rb:35]:
```suggestion
```

Comment (jzbrooks) [reply Formula/swift-format.rb:35]: I'm neither swift package manager or home-brew expert. Could you elaborate a little on why this is helpful?
Per the comment above the line, this should be removed when swift is updated, but I didn't explicitly update swift. Maybe it was implicitly updated with the Xcode toolchain update?

Comment (SMillerDev) [reply Formula/swift-format.rb]: No, it says it can be removed when Swift 5.7 is supported on swift-format.

**Comment by jzbrooks** [comment]
I can't add labels, but I'd appreciate input. Hopefully I can look more into this over the weekend or early next week.

**Comment by Bo98** [comment]
Xcode 14.2 is not compatible with 0.50700.1. It only works with 0.50700.0.

Apple have moved on to Swift 5.8 so this is unlikely to ever be fixed. Swift 5.8 is likely out very soon anyway.

**Comment by jzbrooks** [comment]
I was just discovering some of that myself, but your comment confirms my suspicion that fixing this might be tricky if not impossible. Thanks for the tip.

**Pull request closed without merging**